### ISSUE TITLE
implement APP_HALFTRANS

### ISSUE THREAD

**Issue by lynxlynxlynx** [gemrb/gemrb#671]
![deionnerra](https://user-images.githubusercontent.com/1923867/79050546-9e9c5800-7c22-11ea-88ef-81ae0e6394ae.png)

I never actually noticed this, but it appears there is no transparency, the cutout box of Deionarra here is from my most recent windows gemrb build. I am having teething issues though so you may want to double check.

_Originally posted by @chilvence in https://github.com/gemrb/gemrb/issues/659#issuecomment-612472798_

deion[fc] has 0c 00 0c as the 3 unknown bytes between the extra clown color fields, but none of the colors settings are set. She does have the translucent appearance flags bit on (APP_HALFTRANS). Which we don't use anywhere.

**Comment by lynxlynxlynx** [comment]
Worth checking also if we're just looking up the wrong animation — she has two in resdata.ini

> [14]    // Deionarra
> [2062]   // Deionarra Translucent
and the second one would have a hard time getting used, since it's value is too high (GF_ONE_BYTE_ANIMID only takes into account some other exceptions)

The easiest way to check is just to override the file, swapping the two entries. @chilvence can you give it a go?

**Comment by chilvence** [comment]
Sorry, only I just noticed this while scouring for any new stuff to add to the meta list

I tried switching the ID's around, but the original didn't seem to show any difference, she seems to appear translucent in either case.

It seems to be that the two different appearances of Dieonerra use two different animation ID's, that is all. In the mortuary, she does not move or do anything, while in the negative plane, she can do more stuff. I am not sure how they convert to the ID's in resdata.ini though, in NI it is 57358 vs 54272 depending on which actor...
